### PR TITLE
Update repository URLs

### LINKED ISSUE #567: Replace danderson/metallb with metallb/metallb
We seem to have references to the old location of the repository - `danderson/metallb`. There is a GitHub redirect in place which forwards requests and Git actions to the current repo, however we should probably update these refs to `metallb/metallb` to avoid confusing people.

### PR DESCRIPTION
**Pull request by johananl**
Supersedes #570. Fixes #567.

Opening a new PR as we needed to rebase and I couldn't push to the original branch, likely because the fork no longer exists.

### CODE REVIEW

**Comment by johananl** [comment]
I've tested more or less all the links manually and they seem to work. The only thing I couldn't test is the `_replace()` function in tasks.py:

https://github.com/metallb/metallb/pull/688/files#diff-dc31105b816206de63c47bf42ec2a4d7R302-R304

If we can test this without crafting a release, let's do it and the merge (can we @rata?). If we can't, let's merge as is and fix any problems which may arise as we encounter them.

**Comment by russellb** [comment]
lgtm

**Comment by johananl** [comment]
@rata do you still have any remaining concerns? If not, I'll go ahead and merge.

File: tasks.py [hunk 1]
```diff
@@ -299,9 +299,9 @@ def _replace(pattern):
         newpat = pattern.format("v{}").format(version)
         run("perl -pi -e 's#{}#{}#g' website/content/*.md website/content/*/*.md".format(oldpat, newpat),
             echo=True)
-    _replace("/google/metallb/{}")
-    _replace("/google/metallb/tree/{}")
-    _replace("/google/metallb/blob/{}")
+    _replace("/metallb/metallb/{}")
+    _replace("/metallb/metallb/tree/{}")
+    _replace("/metallb/metallb/blob/{}")
```
Comment (rata) [inline tasks.py:304]: Have you created a release locally to test this? I mean, test that this replace works as expected and there is no missing URL that isn't replaced?

**Comment by johananl** [comment]
> Have you tested locally with `hugo-serve` just in case? Be aware of: #554, though

Yes, I always do on doc changes.

File: tasks.py (hunk 1 above)
Comment (johananl) [reply tasks.py:304]: I wasn't sure it was safe to run the release logic. I'll give it a try.

Comment (johananl) [reply tasks.py:304]: This turned out to be tricky to check/fix. I keep getting unexpected results. Will update once I manage to figure it out.

Comment (johananl) [reply tasks.py:304]: Looks like this works fine.

**Comment by johananl** [comment]
@rata I'd like to add this to `v0.9.5` since without this change we aren't updating the raw manifests anyway.

**Review by rata — APPROVED** [review]
Yes, LGTM.

**Pull request merged**